webpack/components/Table/TableWrapper.js

```diff
@@ -75,6 +75,22 @@ const TableWrapper = ({
     updatePagination(updatedPagination);
   };
 
+  const ShowPageControls = variant => (
```

Comment:
I actually like Andrew's name better than mine:
```suggestion
  const PageControls = variant => (
```